R/codeCheck.R

```diff
@@ -213,7 +212,11 @@ codeCheck <- function(path = ".",
   .emitTimingMessage(" Naming conventions check done...", ptm)
 
   # Check appearance of objects
-  ap <- checkAppearance(gams)
+  capitalExclusionList <- NULL
+  if (file.exists(file.path(path, ".codeCheck.yaml"))) {
+    capitalExclusionList <- read_yaml(file.path(path, ".codeCheck.yaml"))[["capitalExclusionList"]]
```

Comment:
2 Suggestions:

* can we use the name `.codeCheck` instead of ` .codeCheck.yaml`? This seems to me more consistent to all the other config files like `.lintr`, `.Rprofile`, `.buildLibrary` and so on.
* Can you add some information about `.codeCheck` in the documentation of codeCheck so that users are aware of it and know which settings are available?

```suggestion
    capitalExclusionList <- read_yaml(file.path(path, ".codeCheck"))[["capitalExclusionList"]]
```